e2e/app/component/workspace-card.ts

```diff
@@ -170,19 +162,23 @@ export default class WorkspaceCard extends CardBase {
    * @param {boolean} waitForDataPage Waiting for Data page load and ready after click on Workspace name link.
    */
   async clickWorkspaceName(waitForDataPage = true): Promise<string> {
-    const [elementHandle] = await this.asElementHandle().$x(`.//*[${WorkspaceCardSelector.cardNameXpath}]`);
-    await waitForFn(() => {
-      return elementHandle && elementHandle.boxModel() && elementHandle.boundingBox();
-    });
-    const element = BaseElement.asBaseElement(this.page, elementHandle);
-    const name = await getPropValue<string>(elementHandle, 'textContent');
-    await element.click();
+    const workspaceName = await this.getWorkspaceName();
+    const link = new Link(this.page, this.getWorkspaceNameXpath(workspaceName));
+    await link.click();
+    logger.info(`Click workspace name "${workspaceName}" on Workspace card to open workspace.`);
+
     if (waitForDataPage) {
       const dataPage = new WorkspaceDataPage(this.page);
       await dataPage.waitForLoad();
     }
-    logger.info(`Click name "${name}" on Workspace card to open workspace.`);
-    return name;
+    return workspaceName;
```

Comment:
Refactor `clickWorkspaceName` function because click on workspace name fails sometimes. Happened [here](https://app.circleci.com/pipelines/github/all-of-us/workbench/17428/workflows/ec8bf414-fca5-4de1-ac81-c1328a808d77/jobs/193614/tests).